app/models/dialog_field_check_box.rb

```diff
@@ -30,7 +30,7 @@ def normalize_automate_values(automate_hash)
   def refresh_json_value
     @value = values_from_automate
 
-    {:checked => checked?}
+    {:checked => checked?, :read_only => read_only}
```

Comment:
@eclarizio Would it be better to use `read_only?` to ensure we get a true/false value instead of a possible nil?

This would apply to the other sub-classed models as well.